webapp/components/Badges.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div :class="[badges.length === 2 && 'hc-badges-dual']" class="hc-badges">
     <div v-for="badge in badges" :key="badge.id" class="hc-badge-container">
-      <img :title="badge.key" :src="badge.icon | proxyApiUrl" class="hc-badge" />
+      <img :title="badge.key" :src="`/img/badges/${badge.id}.svg`" class="hc-badge" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-30T21:33:24Z" title="Monday, March 30th 2020, 11:33:24 pm +02:00">Mar 30, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,7 +1,7 @@
 <template>
   <div :class="[badges.length === 2 && 'hc-badges-dual']" class="hc-badges">
     <div v-for="badge in badges" :key="badge.id" class="hc-badge-container">
-      <img :title="badge.key" :src="badge.icon | proxyApiUrl" class="hc-badge" />
+      <img :title="badge.key" :src="`/img/badges/${badge.id}.svg`" class="hc-badge" />
```


I decided to move the image files into webapp docker image. Why were they in backend image anyways? :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:31:13Z" title="Wednesday, April 1st 2020, 8:31:13 am +02:00">Apr 1, 2020</time>_

---


that's a good question @roschaefer ...
could we just upload these to `Spaces` as well? then others could more easily reuse while customizing the badges, no? Also, it means smaller docker images

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-04-09T10:56:03Z" title="Thursday, April 9th 2020, 12:56:03 pm +02:00">Apr 9, 2020</time>_

---


The badges are small .svg files. At the moment we don't have a feature to add more badges. So I decided to put them into the docker image.